src/index.js

```diff
@@ -103,18 +109,54 @@ inquirer.prompt(questionnaire).then(async (answers) => {
         }
       );
     }
-    if (isDark && frontEndChoice==='vue') {
-      fs.copyFile(
-        `${currentPath}/themeTemplates/Theme.vue`,
-        `${frontEnd.path}/src/Theme.vue`,
-        (err) => {
-          if (err) {
-            console.log("Error Found:", err);
+
+    //<---------------------------- For TestCases Framework ------------------------------------>
+    if(isTestCasesFramework) {
+      // CYPRESSS
+      if(isCypress) {
+        fs.copyFile(
+          `${currentPath}/uiTests/CypressTests/cypress.config.js`,
+          `${frontEnd.path}/cypress.config.js`,
+          (err) => {
+            if (err) {
+              console.log("Error Found:", err);
+            }
           }
-        }
-      );
+        );
+        fsExtra.copy(
+          `${currentPath}/uiTests/CypressTests/TestScripts`,
+          `${frontEnd.path}`,
+          (err) => {
+            if (err) {
+              console.log("Error Found:", err);
```

Comment:
We should be more explicit while defining errors, as a generic statement tells less about the origin of error. better to have some context.
Error Found:
=>
Error occurred while coping the test script for cypress: ${err}
Error occurred while coping the test script: ${err}

or anything it does

Reply:
For now, updated for these changes only and will create a new issue to update the entire code for errors.